README.md

```diff
@@ -78,37 +78,70 @@ PyMilo is an open source Python package that provides a simple, efficient, and s
 
 
 ## Usage
-### Model preparation 
+Imagine you want to train a `LinearRegression` model based on your dataset `X`, `y`. After splitting the data into train/test segments you would have `X_train` and `y_train` as your training data. So you train your model as follows.
 ```pycon
->>> from sklearn import datasets
```

Comment:
I suggest not deleting these lines. In my opinion, a complete example is better in this case.

Reply:
The reason why I didn't use the complete examples was because using scikit-learn examples have too many features the model JSON representation could be really big in the readme. Confirm here if that's OK, so I will apply this change. @sepandhaghighi @AHReccese

Reply:
I understand your concern. There is no obligation to use scikit-learn examples, but I recommend adding a reproducible example (0 to 100) to `README.md`. For example by using of synthetic small `X` and `y` vectors.

Reply:
Applied. Please re-review.